backend/src/http/handlers.rs

```diff
@@ -59,8 +83,19 @@ pub(super) async fn handle(req: Request<Body>, ctx: Arc<Context>) -> Response {
                 .unwrap()
         }
 
+        "/~metrics" => {
+            register_req!(HttpReqCategory::Metrics);
+            let out = ctx.metrics.gather_and_encode(&ctx.db_pool).await;
+            Response::builder()
+                // TODO: or should this be application/openmetrics-text?
+                .header("Content-Type", "text/plain; version=0.0.4; charset=utf-8")
```

Comment:
This. On a practical level, the former can be easily viewed in the browser while the latter leads the browser to download it as a file. That's why I went with the former. This is also what Prometheus and Opencast use for their endpoints. But yeah, no idea what's "correct" here.

Reply:
I unfortunately don't know either. Where does the latter even come from? It's not mentioned in the IANA MIME type registry. :thinking:

Reply:
So far, I've never seen `application/openmetrics-text`. Just checking a few exporters I use:

```
[lars@lkiesow]/etc/prometheus% curl -si http://127.0.0.1:9093/metrics | grep Content-Type
Content-Type: text/plain; version=0.0.4; charset=utf-8
[lars@lkiesow]/etc/prometheus% curl -si http://127.0.0.1:9219/metrics | grep Content-Type
Content-Type: text/plain; version=0.0.4; charset=utf-8
[lars@lkiesow]/etc/prometheus% curl -si http://127.0.0.1:9113/metrics | grep Content-Type
Content-Type: text/plain; version=0.0.4; charset=utf-8
[lars@lkiesow]/etc/prometheus% curl -si http://127.0.0.1:9100/metrics | grep Content-Type
Content-Type: text/plain; version=0.0.4; charset=utf-8
[lars@lkiesow]/etc/prometheus% curl -si http://127.0.0.1:9090/metrics | grep Content-Type
Content-Type: text/plain; version=0.0.4; charset=utf-8
```

Where did you get `application/openmetrics-text` from?